.cirrus.yml

```diff
@@ -6,9 +6,9 @@ env:
     #### Global variables used for all tasks
     ####
     # Name of the ultimate destination branch for this CI run
-    DEST_BRANCH: "main"
+    DEST_BRANCH: "release-5.26"
     # CI container image tag (c/skopeo branch name)
-    SKOPEO_CI_TAG: "main"
+    SKOPEO_CI_TAG: "v1.13.0"
```

Comment:
I think using a branch name here is slightly better; if we ever tag a v1.13.1 on the stable branch, a branch name reference means that we would keep testing against the relevant Skopeo user.

(Historically `SKOPEO_CI_TAG` was used in `IMAGE_SUFFIX`, hence the name… now that that’s no longer the case, it might make sense to rename this variable to not be confusing — only on `main`, and only if @cevich approves.)

Reply:
Filed https://github.com/containers/image/pull/2083 .